### PR TITLE
HOLD TILL NOV 4 - Update m365d-enable.md

### PR DESCRIPTION
**Pull request by malidgi**
new roles can now enable M365D 
please push the change on Nov 4th, 2021 (not before and not after)

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender/m365d-enable.md [hunk 1]
```diff
@@ -42,7 +42,18 @@ For detailed licensing information, [read the licensing requirements](prerequisi
...
+You must be one of the following roles to turn on Microsoft 365 Defender:
+	•  Global administrator
+	•  Security administrator
+	•  Security Operator
+	•  Global Reader
+	•  Security Reader
+	• Compliance Admin
+	• Compliance Data Admin
+	• Application Admin
+* Cloud Application Admin
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/m365d-enable.md]:
```suggestion
	
	•  Global Administrator
	
	•  Security Administrator
	
	•  Security Operator
	
	•  Global Reader
	
	•  Security Reader
	
	• Compliance Administrator
	
	• Compliance Data Administrator
	
	• Application Administrator
	
	• Cloud Application Administrator
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @malidgi, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@malidgi Gentle reminder

**Comment by malidgi** [comment]
Hey,

Can you please clarify the ask?

Thanks,
Matika

**Comment by JohanFreelancer9** [comment]
@malidgi - you should see an option to **Commit Suggestion** with my suggested changes. Just select or click on it.

**Comment by malidgi** [comment]
Hey 😊

I’m not sure how to do it, can you please share a screenshot?

Thanks,
Matika

**Comment by JohanFreelancer9** [comment]
@yogkumgit - can you please help to get my suggestions committed? I don't have rights to commit them. Thanks.

**Comment by mjcaparas** [comment]
I have committed for Matika.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**